exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,29 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+    var mentor1 = "Daniel";
+    var mentor2 = "Irina";
+    var mentor3 = "Mimi";
+    var mentor4 = "Rob";
+    var mentor5 = "Yohannes";
+
+function helloGreeting(name) {
+    return name;     
+}       
```

Comment:
This function doesn't do anything as is -- all the work is being done by the other function. As per instructions, you should have:
* One function that returns the name in all caps
* One function that prints "HELLO <NAME>"
Note, the second function should call your first function to achieve its goal. See how it's used for age in the example in the readme for this exercise.